integrations/spring/opendal-spring-boot-starter-reactive/pom.xml

```diff
@@ -41,6 +41,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-autoconfigure</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Seems no tests to guard that the configuration work as expected.

@shoothzj what is the plan on this point?

Reply:
@tisonkun I plan to add tests in the future to guard the bean load successfully.